Boards.h

```diff
@@ -560,6 +560,26 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_PWM(p)           (p)
 #define PIN_TO_SERVO(p)         (p)
 
+// Pinoccio Scout
+#elif defined(ARDUINO_PINOCCIO)
+#define TOTAL_ANALOG_PINS       8
+#define TOTAL_PINS              NUM_DIGITAL_PINS // 32
+#define VERSION_BLINK_PIN       23
+#define PIN_SERIAL1_RX          0
+#define PIN_SERIAL1_TX          1
+#define PIN_SERIAL2_RX          13
+#define PIN_SERIAL2_TX          14
+#define IS_PIN_DIGITAL(p)       (((p) >= 2) && ((p) <= 8)) || (((p) >= 24) && ((p) <= 31))
```

Comment:
Okay I found the pinout for the Pinoccio. They did tie the usb serial to digital pins 0 and 1. The number at the end of PIN_SERIAL aligns with the Serial object, so PIN_SERIAL1 is for Serial1, PIN_SERIAL2 is for Serial2. You should remove lines 568 and 569 and change the following 2 lines to PIN_SERIAL1 (so you'll only map to pins 13 and 14 for now since the default Serial object is reserved for Firmata communication). At some point I'll open it up when using Firmata via a non-serial channel such as Ethernet or WiFi.